src/test/resources/scenarios/dashboard.feature

```diff
@@ -0,0 +1,15 @@
+Feature: Dashboard Page Tests
+
+  Background:
+#    Given Load TestCases for "login"
```

Comment:
no code should be left in comments.

Reply:
Ok.
Deleted all commented unwanted code, now we can baseline the version as 1.0